releasing/docker-generate.sh

```diff
@@ -0,0 +1,289 @@
+FROM ubuntu:$UBUNTU_VERSION
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV TZ UTC
+
+RUN apt-get update
+
+# Install basic tools
+RUN apt-get install -y          \\
+    curl                        \\
+    file                        \\
+    git                         \\
+    gnupg                       \\
+    lsb-release                 \\
+    software-properties-common  \\
+    wget
+
+EOF
+
+    # Install compiler
+    # --------------------------------------------------------------
+    case $UBUNTU_VERSION in
+        xenial)
+            cat >> ubuntu-${UBUNTU_VERSION}/Dockerfile <<EOF
+# Get a newer GCC and flex version
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test; apt-get update
+RUN apt-get install -y 	\\
+    bison               \\
+    build-essential     \\
+    flex  		\\
+    g++-7 		\\
+    gcc-7
+
+RUN ln -sf /usr/bin/gcc-7 /usr/bin/gcc
+RUN ln -sf /usr/bin/g++-7 /usr/bin/g++
+
+# Link libstdc++ statically
+ENV BAZEL_LINKOPTS "-static-libstdc++:-lm"
+ENV BAZEL_LINKLIBS "-l%:libstdc++.a"
+EOF
+            ;;
+        *)
+	    cat >> ubuntu-${UBUNTU_VERSION}/Dockerfile <<EOF
+
+# Install compiler
+RUN apt-get install -y  \\
+    bison               \\
+    build-essential     \\
+    flex                \\
+    g++                 \\
+    gcc
+
+EOF
+            ;;
+    esac
+
+    # Install Bazel
+    # --------------------------------------------------------------
+    cat >> ubuntu-${UBUNTU_VERSION}/Dockerfile <<EOF
+
+# Install bazel
+RUN \\
+    wget --no-verbose "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel_${BAZEL_VERSION}-linux-x86_64.deb" -O /tmp/bazel.deb; \\
+    dpkg -i /tmp/bazel.deb || true; \\
+    apt-get -f install -y
+
+EOF
+
+done
+
+# ==================================================================
+# Generate the docker files for centos versions
+# ==================================================================
+for CENTOS_VERSION in 6 7 8; do
+    # Install basic tools
+    # --------------------------------------------------------------
+    cat > centos-${CENTOS_VERSION}/Dockerfile <<EOF
+FROM centos:$CENTOS_VERSION
+
+# Install basic tools
+RUN yum install -y  \\
+    file            \\
+    git             \\
+    redhat-lsb      \\
+    tar             \\
+    wget
+
+EOF
+
+    # Install compiler
+    # --------------------------------------------------------------
+    case $CENTOS_VERSION in
+        6|7)
+            cat >> centos-${CENTOS_VERSION}/Dockerfile <<EOF
+# Link libstdc++ statically so people don't have to install devtoolset-8
+# just to use verible.
+ENV BAZEL_LINKOPTS "-static-libstdc++:-lm -static-libstdc++:-lrt"
+ENV BAZEL_LINKLIBS "-l%:libstdc++.a"
+
+# Get a newer GCC version
+RUN yum install -y --nogpgcheck centos-release-scl
+RUN yum install -y --nogpgcheck devtoolset-8
+
+RUN yum install -y bison flex
+
+# Build a newer version of flex
+RUN \\
+    wget --no-verbose "https://github.com/westes/flex/files/981163/flex-2.6.4.tar.gz" && \\
```

Comment:
Would it make sense to define a `FLEX_VERSION=2.6.4` variable similarly in settings.sh?

Reply:
Probably not, the CI system doesn't use it -- only the release builders.